### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by jmcarp**
* [x] Implement fullcalendar
* [x] Add filter sidebar
* [ ] Write unit tests

### CODE REVIEW

**Comment by noahmanger** [comment]
It looks like we're missing filters for committee hearings / meetings type of events like these:

image

Or am I missing something?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js: "0.12"
```
Comment (ccostino) [inline .travis.yml]: Shouldn't we change this to a more recent Node version?

File: package.json [hunk 2]
```diff
@@ -1,18 +1,42 @@
 {
+  "scripts": {
+    "test": "karma start",
+    "test-single": "karma start --single-run --browsers PhantomJS"
+  },
   "dependencies": {
     "ace-builds": "1.2.2",
     "component-sticky": "1.0.0",
-    "fec-style": "1.4.1",
+    "fec-style": "git://github.com/18f/fec-style#styling-calendar",
```
Comment (ccostino) [inline package.json]: This dependency is now pointing to a specific branch as opposed to a version?

Comment (noahmanger) [inline package.json]: Oh yeah, we'll want to change this when we merge in https://github.com/18F/fec-style/pull/238 and bump the version number.

Comment (ccostino) [inline package.json]: @noahmanger:  Should that PR be reviewed as well and merged in first?

Comment (noahmanger) [inline package.json]: Ah, yes it should. It was assigned to @jmcarp but I don't care who does it.

**Comment by ccostino** [comment]
@jmcarp and I talked a bunch about this PR and decided that more tests would come at a later time when things settle down a bit more and there's more concrete use-cases to go on.  We're merging what has been done so far.

**Pull request merged**